src/opentracing_external.cpp

```diff
@@ -21,18 +24,20 @@ std::tuple<std::shared_ptr<ot::Tracer>, std::shared_ptr<TraceEncoder>> makeTrace
     const TracerOptions &options) {
   auto maybe_options = applyTracerOptionsFromEnvironment(options);
   if (!maybe_options) {
-    std::cerr << "Error applying TracerOptions from environment variables: "
-              << maybe_options.error() << std::endl
-              << "Tracer will be started without options from the environment" << std::endl;
+    std::ostringstream message;
```

Comment:
Could this be avoided with additional parameters in log methods?
Ideally we'd be using fmtlib or something, but that could be iterated on in the future.

Reply:
Yes. I'd prefer to defer the whole question of a better logging interface, though.